encoding/preparation.md

```diff
@@ -250,7 +250,83 @@ such as Intel's [SVT-AV1][] will also not be included.
 
 ## Audio Codecs
 
-TODO: FLAC, AAC, maybe mention opus
+### Audio formats and how to handle them
+
+Depending on the source you'll be working with,
+you may encounter many different audio formats.
+
+On Blu-Rays, 
+you will most likely find audio encoded losslessly, 
+in the form of either DTS-HD Master Audio, Dolby TrueHD, or PCM. 
+DTS-HD MA and Dolby THD are proprietary codecs that use lossless compression, 
+while PCM is simply raw, uncompressed PCM data.
+The usual way handle these formats is to reencode them to other formats - 
+either lossless or lossy, depending on your taste.   
```

Comment:
Is the issue here that it's a keyboard dash/hyphen and not an em-dash? I was under the impression it ignores trailing whitespace at the end of lines. 

And for this example, this would still be a problem if it does. It would make the word after the hyphen not have a space separating it from the hyphen. unlike the word "formats" and the hyphen. I'd recommend using an em dash `—` with no whitespace (can't be at the end of a line) on either side.

This would result in `...is to reencode them` \N `to other formats—either lossless or lossy, depending on your taste.`

Reply:
What I was referring to is the trailing three spaces in this line. You don't see them by default, but I have https://addons.mozilla.org/de/firefox/addon/render-whitespace-on-github/?src=search for that purpose since whitespace is frequently an issue when reviewing pull requests and code on GitHub..